### PR TITLE
[processor/spanmetricsprocessor] Add ms unit to latency buckets

### PR DESCRIPTION
**Pull request by Yamakaky**
**Description:** Define the `latency` metric unit as milliseconds for `spanmetricsprocessor`.

**Testing:** unit tests

**Documentation:** the metric was already documented as being in milliseconds.

### CODE REVIEW

File: processor/spanmetricsprocessor/README.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ Aggregates Request, Error and Duration (R.E.D) metrics from span data.
 **Request** counts are computed as the number of spans seen per unique set of dimensions, including Errors.
 For example, the following metric shows 142 calls:
 ```
-calls{http_method="GET",http_status_code="200",operation="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET"} 142
+calls_total{http_method="GET",http_status_code="200",operation="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET"} 142
```
Comment (dmitryax) [inline processor/spanmetricsprocessor/README.md:17]: This is unrelated change, right? Can you please submit another PR?

Comment (Yamakaky) [reply processor/spanmetricsprocessor/README.md:17]: Sure

**Review by dmitryax — APPROVED** [review]
LGTM. cc @albertteoh as code owner

**Comment by Yamakaky** [comment]
Should I update the changelog?

**Comment by dmitryax** [comment]
Yes, I believe it should be mentioned in the changelog. please create an `enhancement` entry in `unreleased` directory

**Review by albertteoh — APPROVED** [review]
LGTM.

Forgive my ignorance, but what are the use cases for the optional metric units in the telemetry data itself? My understanding, at least for prometheus, is that units are not stored with the metric data but, instead, are carried in the metric name itself like `http_request_duration_seconds`.

**Comment by Yamakaky** [comment]
Hum, relevant references:
- https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#units
- https://prometheus.io/docs/practices/naming/

Based on that, I think the metric should be renamed to something like `span.duration` and use seconds as units. What do you think?
